topology/The University of Montana/FACILITY.yaml

```diff
@@ -0,0 +1 @@
+ID: 10213
```

Comment:
I believe that this should be auto-generated as well. Can you try deleting this file?

Reply:
file removed